packages/ui-components/package.json

```diff
@@ -27,18 +27,22 @@
     "recharts": "^1.6.0",
     "semantic-ui-css": "^2.4.1",
     "semantic-ui-react": "^0.88.1",
-    "styled-components": "^4.4.1"
+    "styled-components": "^5.0.1"
   },
   "devDependencies": {
-    "@storybook/addon-actions": "^5.2.5",
-    "@storybook/addon-knobs": "^5.2.5",
-    "@storybook/addon-links": "^5.2.5",
-    "@storybook/react": "^5.2.5",
+    "@babel/compat-data": "7.8.0",
```

Comment:
this was added to fix https://github.com/facebook/create-react-app/issues/8680

see also: https://stackoverflow.com/questions/60790726/could-not-find-plugin-proposal-numeric-separator-in-angular-app

Reply:
Ah, I stumbled on this issue too 2 weeks ago. As said in the issue, it's fixed in 3.4.1. So I would remove @babel/compat-data and the resolution below.

What I did was: `rm **/node_modules/** (also the node_modules inside packages) && yarn cache clean && yarn install`, and after that it worked.